### PR TITLE
Update cont aa1 to master (Update CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by schettn**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the contribution seems to be a bit odd.

**What is the new behavior (if this is a feature change)?**
- The contribution file has has been updated to include guidelines to improve successful 
collaboration.

**Other information**:
-  🐍

### CODE REVIEW

**Review by Aichnerc — APPROVED** [review]
LGTM

**Review by pinterid — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,45 @@
...
+
+## [](#pull-requests)Pull Requests
+
+-   Each pull request should only follow one topic
+-   You must follow the guidelines / template
+-   Each pull request must answer one more more issues
+-   Each pull request must have at least one reviewer
+-   Each pull request must have at least one assigned developers
+-   Each pull request must have a label
+-   Each pull request should be merged by a reviewer
```
Comment (kleberbaum) [inline CONTRIBUTING.md]: We should have a meeting about this line.
```suggestion
-   Each pull request should be merged by the maintainer
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,9 +1,45 @@
...
+-   Include reference to issue if one exists
+-   All commits must be verified
+-   No commits, under no circumstances, should be directly pushed to master (deadly)
+-   Each commit should only serve one purpose
+
+## [](#pull-requests)Pull Requests
+
+-   Each pull request should only follow one topic
+-   You must follow the guidelines / template
+-   Each pull request must answer one more more issues
```
Comment (kleberbaum) [inline CONTRIBUTING.md]: Not really needed anymore.
```suggestion
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**